Update order-status.md

larsolofsson opened the pull request:
I have corrected errors in the text and deleted Scenario F Over Shipment.
If some reels are taken from stock, then they are normally included in Produced quantity. It would mean that Produced and Shipped quantities would be the same when ShipmentCompleted, so this isn't really a new scenario.

We also have other scenarios, e.g. the products can be invoiced before they are shipped. We have a major difference in this scenario compared to existing scenarios, orderStatus=Completed is after ShipmentCompleted and not after the issue of the invoice as in existing scenarios.
How do we communicate the status that the order line items are invoiced when invoicing is done before shipments? Or is it enough to report only an Invoiced quantity?
Should we add this scenario?

We should also discuss if we need more quantity types. E.g. a merchant is selling sheeted paper from stock, would he use quantity type Produced in the API response, when he has allocated stock items to the customer order? The merchant is the supplier in this case.

patricekrakow commented:
Hello @larsolofsson, you can noticed that, in the meantime, I have corrected all the `"orderLineItemNumber": "1"` to `"orderLineItemNumber": 1` as the field `orderLineItemNumber` has `type: integer`. You can change all of them using the "find and replace" functionality of your editor. **_Important:_** You do not need to create a new PR for this extra change in your `order-status.md` file, you can just push a new commit in your `feature/Lars-order-status_1.1.0` branch and this new commit will be (automatically) part of this PR.

patricekrakow reviewed (changes requested):
You should change `"orderLineItemNumber": "1"` to `"orderLineItemNumber": 1` as the field `orderLineItemNumber` has `type: integer`. This correction has been (already) merged into the `develop` branch, so it's important you do not remove it by merging this PR.

larsolofsson commented:
Thank you Patrice,

I have now 2 commits on PR #31.

I have changed all instances of "orderLineItemNumber": "1", to "orderLineItemNumber": 1,

 

I now understand after your comment how to work with commits when changes are done in a PR.

 

In PR #30 the changed file was stored in a wrong folder. I created a new branch and copied the file into the right folder 1.1.0 and created a new PR #31. I closed PR #30.

Could I do this in another way?

Would it be possible to just copy the file from the wrong folder into folder 1.1.0, delete the wrong folder and make a commit and push it to update PR #30?

 

 

Kind regards

Lars

 

 

From: Patrice Krakow ***@***.*** 
Sent: den 24 maj 2021 10:39
To: papinet/papiNet-API
Cc: Lars Olofsson; Mention
Subject: Re: [papinet/papiNet-API] Update order-status.md (#31)

 

Hello @larsolofsson <https://github.com/larsolofsson> , you can noticed that, in the meantime, I have corrected all the "orderLineItemNumber": "1" to "orderLineItemNumber": 1 as the field orderLineItemNumber has type: integer. You can change all of them using the "find and replace" functionality of your editor. Important: You do not need to create a new PR for this extra change in your order-status.md file, you can just push a new commit in your feature/Lars-order-status_1.1.0 branch and this new commit will be (automatically) part of this PR.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/papinet/papiNet-API/pull/31#issuecomment-846882726> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/APAU5RMOWMYILI6RLCEECUTTPIGB3ANCNFSM45IXTS5Q> .  <https://github.com/notifications/beacon/APAU5RJKR56TI34SHCENNJDTPIGB3A5CNFSM45IXTS52YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOGJ5GPJQ.gif>

patricekrakow reviewed (approved):
Great ;-)
